langchain4j-core/src/main/java/dev/langchain4j/agent/tool/Tool.java

```diff
@@ -36,4 +36,17 @@
      * @return description of the tool.
      */
     String[] value() default "";
+
+    /**
+     * Return behavior of the tool.
+     * - If {@link ReturnBehavior#TO_LLM} is used (default), the value returned by the tool is sent back to the LLM for further processing.
+     * - If {@link ReturnBehavior#DIRECT} is used, the value returned by the tool is sent to the LLM to allow it to invoke more tools if necessary.
```

Comment:
I find the name `DIRECT` confusing in this case, but I have to admit that I can't think of anything better...